README.md

```diff
@@ -26,38 +40,28 @@ pip install -r requirements.txt
 
 The virtual environment can be replaced with another local setup.
 
-### Alternate TSP version
-
-Above, `API.yaml` has been the manually documented TSP version up until now.
+### Generate the API
 
-* That version is the one shown by the default [github pages][tspGhPages].
-* That version does not fully match what [Incubator][incubator]'s trace-server currently supports.
-  * Some TSP endpoints differ or are missing in trace-server, despite its use to help refine the protocol.
-  * That trace-server being the TSP reference implementation, among potential other ones.
-* Aligning the TSP documented that way with trace-server's own (reference) endpoints is a work in progress.
+Swagger has recently been added to the Trace Compass trace-server (reference implementation).
 
-In the meantime, Swagger has recently been added to trace-server.
-
-* Below is how to generate the alternate TSP version, according to Swagger in trace-server.
-* This alternate version is to be gradually augmented based on what has been manually documented through `API.yaml`.
-* [Swagger github pages][tspSwagger] show this alternate version to grow.
-* `API.yaml` should gradually become the wanted version of the TSP, as this alternate one shows the reference TSP.
-* Such reference and future (forecasted) TSP versions will be documented accordingly as these yaml files evolve.
+* Below is how to generate the TSP version, according to Swagger in trace-server.
+* The generated TSP should match the current supported version of the TSP. Any differences may be pushed for review.
+* `API.yaml` shows now the current supported version of the TSP.
 
 1. Import all [TC][tracecompass] and [incubator][incubator] projects in Eclipse; branch, Target Platform and API Baseline set to `master`.
 1. Open `traceserver.product` file in plug-in `org.eclipse.tracecompass.incubator.trace.server.product`
 1. Click on the `Run` button on the top right corner of the opened `traceserver.product`
 1. Browse [to here][apiyaml] ([swagger][swagger]) or so to generate server's TSP.
-1. Bring the resulting file over; e.g.: `mv ~/Downloads/openapi.yaml .`
+1. The resulting is stored in the user's Downloads directory; e.g.: `~/Downloads/openapi.yaml`
 1. Update the latter with its license information and remove extra information: `./openapi.py`
```

Comment:
This .py script also likely needs updates in order to verify this whole change. Maybe you had it done locally so you could test these updated steps. (Include in this commit?)

Reply:
This PR, AFAIK, aims at renaming openapi.yaml to API.yaml, and the previous API.yaml to API-proposed.yaml. Hence openapi.py requiring a corresponding update too. (Right?)

Reply:
I added a step to copy the generated openapi.yaml file to this repo before running openapi.py. So, it still works. 
Maybe I need to be clearer which files to diff (API.yaml and openapi.yaml). Also, that the matching updates should be transferred to API-proposed.yaml) and then at the end openapi.yaml can be removed when not needed anymore.